### PR TITLE
[#272] Remove exception on empty list in batch invocation

### LINKED ISSUE #272: ModelClient.batch fails on empty list
If I pass an empty list to batch function, I expect to get an empty response, but i got an exception.
```
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>500 Internal Server Error</title>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error and was unable to complete your request.  Either the server is overloaded or there is an error in the application.</p>
```
I guess request even shouldn't be sent, model client can return empty list in such case, without passing empty parameters to the model.

### PR DESCRIPTION
**Pull request by leshchanka-aliaksandr**
This closes #272

### CODE REVIEW

**Comment by kirillmakhonin** [comment]
@leshchanka-aliaksandr I think it would be better to add verification on server side

File: legion/tests/test_pyserve.py [hunk 1]
```diff
@@ -190,6 +190,17 @@ def test_model_invoke_summation_in_batch_mode(self):
             self.assertIsInstance(response, list, 'Result is not a list')
             self.assertListEqual(response, expected_answer, 'Invalid answer')
 
+    def test_model_invoke_summation_with_empty_list_in_batch_mode(self):
+        with ModelServeTestBuild(self.MODEL_ID, self.MODEL_VERSION,
+                                 create_simple_summation_model_by_df) as model:
+            parameters = [{} for _ in range(10)]
```
Comment (kirillmakhonin) [inline legion/tests/test_pyserve.py]: It is not empty list. It is list of empty elements...

**Comment by leshchanka-aliaksandr** [comment]
test this please

**Comment by kirillmakhonin** [comment]
bot, test app

**Comment by aliaksandr-d** [comment]
```
TASK [legion_enclave : Generate S3 access policy documents] ********************
ok: [localhost] => (item=airflow_s3_access_policy)
failed: [localhost] (item=event_collector_s3_access_policy) => {"changed": false, "item": "event_collector_s3_access_policy", "msg": "AnsibleUndefinedVariable: 'collector' is undefined"}
```

**Comment by aliaksandr-d** [comment]
bot, test app

**Pull request merged**